lib/deployment-tasks.js

```diff
@@ -17,7 +17,7 @@ var _ = require('lodash');
 /**
  * Creates a Pull Image Task to download the Docker Image currently in use by the provided service
  * @param  {module:AWSTools~ContainerService} containerService - container service to check for currently deployed services.
- * @param  {module:AWSTools~Registry} registry - Remove ECR Registery to pull from
+ * @param  {module:AWSTools~Registry} registry - Remove ECR Registry to pull from
```

Comment:
While you're fixing spelling, I think this is the `Remote ECR Registry` rather than the `Remove ECR Registry`  :)